util/reggen/validate.py

```diff
@@ -1151,115 +1157,104 @@ def validate_multi(mreg, offset, addrsep, width, top):
     if error > 0:
         return (error, 0)
 
-    bused = gen[3]
-    max_rval = (1 << width) - 1
+    # multireg attributes
     cname = mreg['cname']
-    bpos = 0
-    inum = 0
+    template_reg = {
```

Comment:
If this data structure is used (or exported), then, could you also consider to define this as `OrderedDict` format rather than `dict`?

Reply:
yes good call.